haunted-houses/src/main/java/cz/muni/fi/pa165/hauntedhouses/dao/HouseDao.java

```diff
@@ -1,4 +1,49 @@
 package cz.muni.fi.pa165.hauntedhouses.dao;
 
+import cz.muni.fi.pa165.hauntedhouses.model.House;
+
+import java.util.List;
+
+/**
+ * @author Petr Vitovsky
+ */
 public interface HouseDao {
+
+    /**
+     * Adds new House to the database
+     * @param house New House
+     */
+    void create(House house);
+
+    /**
+     * Searches database for a House with a specific id
+     * @param id Id of a House
+     * @return House with the given id
+     */
+    House getHouseById(long id);
```

Comment:
Napdla mi ešte jedna vec. Nebolo by dobré zjednotiť nazvy tých CRUD operácií? Buď by všetko obsahovalo slovo "house" (createHouse, updateHouse,getHouseById,...) alebo nič (create, update, getById,...)? Zaujímal by ma váš názor.

Reply:
Asi by to tak bylo lepší, jsem pro variantu s názvem entity.

Reply:
Pojmenování sjednoceno.

Reply:
navrhujem ich tam nedat, nakolko uz objekt ktory vytvoris sa bude nazyvat napr ako hauseDao